src/shared/seccomp-util.c

```diff
@@ -21,6 +21,7 @@
 #include "set.h"
 #include "string-util.h"
 #include "strv.h"
+#include "env-util.h"
```

Comment:
we try to keep includes ordered alphabetically, see https://systemd.io/CODING_STYLE/